src/deluge/gui/ui/keyboard/layout/chord_keyboard.cpp

```diff
@@ -137,4 +143,14 @@ void KeyboardLayoutChord::drawChordName(int16_t noteCode, const char* chordName,
 		display->setScrollingText(fullChordName, 0);
 	}
 }
+
+bool KeyboardLayoutChord::allowSidebarType(ColumnControlFunction sidebarType) {
+	if ((sidebarType == ColumnControlFunction::CHORD) || (sidebarType == ColumnControlFunction::DX) ||
+	    // TODO, when scales for chord keyboard are implemented, add this back in
+	    (sidebarType == ColumnControlFunction::SCALE_MODE)) {
```

Comment:
I figured these 3 were the ones it should skip for now. Does that seem right? `CHORD` for sure, `SCALE_MODE` until  the chord keyboard supports scales, and would `DX` be good to skip too? Any others?

Reply:
I think the DX sidebar should stay, it's skipped unless you're on a DX7 patch anyway